Pharmacy Database Project-csharp version/Domain/Repositories/Interface/IBatchRepository.cs

```diff
@@ -0,0 +1,10 @@
+using Domain.Models;
+
+namespace Domain.Repositories.Interface;
+
+public interface IBatchRepository
```

Comment:
we should make all the Repositories implement the generic Repository interface, I mean something like this:

```suggestion
public interface IBatchRepository : IRepository<BatchDomain, int>
```